demos/demo_reactive_planners_solo12_step_adjustment_walk.py

```diff
@@ -0,0 +1,317 @@
+"""Demo file for running the reactive planner on SOLO 12
+The logic is the same as the demo_reactive_planners_solo12_step_adjustment_walk.ipynb
+This file is used for ROS 2. An example of the usage of this class is here: 
```

Comment:
Why do you say it is used for ROS2? do you depend on ROS2? do you use rclcpp or rclpy?

Reply:
Yes, we have a ROS 2 node that uses demo_reactive_planners_solo12_step_adjustment_walk.py and demo_reactive_planners_solo12_step_adjustment_walk.cpp. We use both rclcpp and rclpy. But I agree, both the c++ and Python reactive planner demo files do not need to be used in a ROS2 node.